index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
If we place this file in /docs, we would have to duplicate our README. I thought it was preferrable this way.